src/app/dataset/[slug]/page.tsx

```diff
@@ -63,6 +62,30 @@ export default function Page({
             href: '/dataset/Badplatser',
         },
     ]
+    let eventsRespond: EventAPISchema | undefined
+    let parsedPageData: datasetSchema | null = null
+    try {
+        if (slug) {
+            const pageData = datasetSchema.safeParse(await getDataset(datasetWithSpace(decodeURI(slug))))
```

Comment:
We should handle this as an actual slug, right now it's not. It's case sensitive at the moment and spaces should be converted to `-` 

So the slug of `Social Media Mentions` should be `social-media-mentions`